scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -630,6 +600,48 @@ static struct task_ctx *lookup_task_ctx(struct task_struct *p)
 	return taskc;
 }
 
+int save_gpu_tgid_pid() {
```

Comment:
Probably should explicitly mark this as void as sometimes the verifier gets confused. Especially since this is a global.
```suggestion
int save_gpu_tgid_pid(void) {
```